Assets/Scripts/Structs/PlayerData.cs

```diff
@@ -17,14 +18,18 @@ public class PlayerData
         public int ScoreMultiplier 
         {
             get => _scoreMultiplier;
-            set => _scoreMultiplier = Mathf.Max(value, 0);
+            set => _scoreMultiplier = Mathf.Max(value, 1);
         }
         public float IntoxicationLevel
         {
             get => _intoxicationLevel;
             set => _intoxicationLevel = Mathf.Clamp(value, 0.0f, 1.0f);
         }
-
+        public int HealthPoints 
+        {
+            get => _healthPoints;
+            set => _healthPoints = Mathf.Max(value, 0);
```

Comment:
I think that the default value should be something like 3 instead of 0, because if there is a problem, we still want the user to be able to play

Reply:
The default is already 3, this is just so that the health doesn't go bellow 0 in case there is a bug